libmscore/input.cpp

```diff
@@ -60,7 +60,7 @@ Fraction InputState::tick() const
 
 ChordRest* InputState::cr() const
       {
-      return _segment ? toChordRest(_segment->element(_track)) : 0;
+      return _segment && _track >= 0 ? toChordRest(_segment->element(_track)) : 0;
```

Comment:
That `_track >= 0` requires additional comments in the code, imho. Please, mention in the comments, why you decided to change the semantic of the pretty fundamental method?

Reply:
OK.  Pretty sure there was no reason other than protection against a crash if _track ends up being -1 at the point where this called, but I'll review.

Reply:
After investigation, I confirm the possibility that navigation through frames or other elements not attached to any particular staff can leave track at -1, so the above is needed to avoid a potential crash.  It has no other effect, in all other cases track is >= 0.  But I've added a comment to that effect in a commit I'll push soon.